warehouse/static/js/warehouse/utils/unsupported_browser.js

```diff
@@ -0,0 +1,16 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default (fn) => {
+  return navigator.appVersion.includes("MSIE 10");
```

Comment:
This is simple enough that I don't think it needs to be in a separate file, let's just move it inline instead.

Reply:
I think Content-Security-Policy (CSP) prohibits such inlining. In initial development of this fix we kept running into all kinds of CSP violations for JS and CSS.

Reply:
Youre right, but I meant inline in `doc-ready.js`,  not in the HTML.